index.js

```diff
@@ -36,6 +46,15 @@ Babel.prototype.constructor = Babel;
 Babel.prototype.extensions = ['js'];
 Babel.prototype.targetExtension = 'js';
 
+Babel.prototype.write = function(readTree, destDir) {
+  var self = this;
+  return Filter.prototype.write.call(this, readTree, destDir).then(function() {
+    if (self.exportModuleMetadata) {
+      fs.writeFileSync(destDir + path.sep + 'dep-graph.json', JSON.stringify(self.moduleMetadata, undefined, 2));
```

Comment:
> I'm not really seeing the value as this file must change if you are adding or removing files. Downstream this is hashed and is used as the initial cache key to determine if the overall dependency graph has changed. If none of the dep-graphs have changed we just symlink everything over that was in the previous build.

sure.

But the second part of my comment still appears valid, the output should be stable correct?